include/cinder/Log.h

```diff
@@ -82,9 +82,6 @@ class Logger {
 
 	virtual void write( const Metadata &meta, const std::string &text ) = 0;
 
-	template<typename LoggerT, typename... Args>
-	static std::shared_ptr<LoggerT> makeLogger( Args&&... args );
```

Comment:
That makes sense to me, but they can still use `std::make_shared<LoggerT>()` directly as the constructor is public. This did get me thinking, we should probably make Logger inherit from `ci::Noncopyable` at this point, to indicate that they should be created on the heap. I'll add that to this PR.